File: apps/claimly-server/src/APIs/Claim/Dtos/ClaimCreateInput.cs
namespace Claimly.APIs.Dtos;

public class ClaimCreateInput
{
    public double? Amount { get; set; }

    public ClaimStatus? ClaimStatus { get; set; }

    public DateTime CreatedAt { get; set; }

    public Customer? Customer { get; set; }

    public DateTime? Date { get; set; }

    public string? Description { get; set; }

    public string? Id { get; set; }

    public Policy? Policy { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: apps/claimly-node-js-admin/src/api/claim/ClaimCreateInput.ts
import { ClaimStatusWhereUniqueInput } from "../claimStatus/ClaimStatusWhereUniqueInput";
import { CustomerWhereUniqueInput } from "../customer/CustomerWhereUniqueInput";

export type ClaimCreateInput = {
  amount?: number | null;
  claimStatus?: ClaimStatusWhereUniqueInput | null;
  customer?: CustomerWhereUniqueInput | null;
  date?: Date | null;
  description?: string | null;
};
